dask/array/creation.py

```diff
@@ -135,3 +137,40 @@ def arange(*args, **kwargs):
         elem_count += bs
 
     return Array(dsk, name, chunks, dtype=dtype)
+
+
+@wraps(np.indices)
```

Comment:
I basically got this from copy-n-pasting something from `dask.array.core`. Is there any value in having this for Dask beyond getting the docstring? If not, was planning to replace the docstring given our parameters differ.